apps/www/data/products/database/highlight-cards.ts

```diff
@@ -34,16 +34,10 @@ const cards: { [name: string]: CardProps } = {
     ],
     ctas: [
       {
-        label: 'Get Early Access',
-        href: 'https://forms.supabase.com/enterprise',
-        target: '_blank',
-        type: 'primary',
-      },
-      {
-        label: 'Learn more',
+        label: 'Get Started',
```

Comment:
```suggestion
        label: 'Learn more',
```

to keep it consistent with the branching section